### ISSUE TITLE
Unexpected External value as type specifier, expected string or type

### ISSUE THREAD

**Issue by nomagick** [Koromix/koffi#144]
Issue with node 20.12.0 (macos arm64): 
`Unexpected External value as type specifier, expected string or type`

Node 12.11.1 works fine.

To recreate:
```bash
nvm use 20.12

node
```

```javascript
koffi = require('koffi');
opaqueType = koffi.opaque();
pointer = koffi.pointer(opaqueType);
```

**Comment by chrbongardt** [comment]
This also happens on windows x64 as well. Happens with node 20.12+

**Comment by Koromix** [comment]
It is caused by what I think is a regression in Node.js itself regarding the tagging of externals.

I'll signal them; in the meantime I've **released Koffi 2.8.1** which works around this issue and works ;)
